SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetDriver.h

```diff
@@ -159,6 +159,8 @@ class SPATIALGDK_API USpatialNetDriver : public UIpNetDriver
 
 	void DelayedSendDeleteEntityRequest(Worker_EntityId EntityId, float Delay);
 
+	virtual void FinishDestroy() override;
```

Comment:
Where `PostInitProperties` is defined, we have comments saying `Begin UObject Interface`. You can move FinishDestroy to be up there as well.

Reply:
Good idea.